package.yaml

```diff
@@ -91,6 +91,7 @@ dependencies:
   - flow
   - filepath
   - fsnotify
+  - Glob
```

Comment:
I hate that this package is capitalized :unamused:

Reply:
I know, right?! Especially for something that's fairly widely used 😭